syntax = "proto3";

import "nanopb.proto";
import "BasicTypes.proto";
import "CryptoCreateTransactionBody.proto";
import "CryptoTransferTransactionBody.proto";

message HederaTransactionBody {
    uint64 transactionFee = 3;
    oneof data {
        HederaCryptoCreateTransactionBody cryptoCreateAccount = 11;
        HederaCryptoTransferTransactionBody cryptoTransfer = 14;
    }
}
